docs/releases/v2019.1.rst

```diff
@@ -0,0 +1,229 @@
+
+See the :ref:`Build configuration <user-site-build-configuration>` section for details.
+
+Hoodselector: Geolocation Mode
+==============================
+
+Hoodselector is a new concept for continuous reevaluation of the node's domain. As a first step we support its geolocation feature. It can be used to automatically switch nodes during their runtime between domains.
+
+See the :doc:`../package/gluon-hoodselector` documentation for details.
+
+
+x86 images support firstboot
+============================
+
+x86 images are now using squashfs instead of ext4 and can now have their configuration reset by using ``firstboot``.
+
+
+Bugfixes
+********
+
+* Fixes passwordless SSH access when gluon-authorized-keys was used without gluon-setup-mode. (`#1777 <https://github.com/freifunk-gluon/gluon/issues/1777>`_)
+
+- Fixes cross-domain leakage of respondd data by not joining the link-local multicast group on br-client. Nodes now only listen on ``[ff05::2:1001]:1001`` for respondd queries on br-client. This may require an update to your respondd querying tooling. (`#1701 <https://github.com/freifunk-gluon/gluon/issues/1701>`_)
+
+
+Site changes
+************
+
+When updating a site configuration from Gluon 2018.2.x, the following changes
+must be made:
+
+site.mk
+=======
+
+- We now require the ``GLUON_DEPRECATED`` variable to be set to decide how to handle the image generation for deprecated devices. (`#1745 <https://github.com/freifunk-gluon/gluon/pull/1745>`_)
+
+- The variable ``DEVICES``, that controls which devices to build images for, has been renamed to ``GLUON_DEVICES``. (`#1686 <https://github.com/freifunk-gluon/gluon/pull/1686>`_)
+
+site.conf
+=========
+
+- The ``mesh.batman_adv.routing_algo`` option is now required, when the batman-adv routing protocol is used. (`#1622 <https://github.com/freifunk-gluon/gluon/pull/1622>`_)
+
+- The options ``wifi*.basic_rates`` and ``wifi*.supported_rates`` have been removed, as the legacy 802.11b rates are now disabled by default. (`#1716 <https://github.com/freifunk-gluon/gluon/pull/1716>`_)
+
+
+Internals
+*********
+
+Debug Build-Flag
+================
+
+Setting ``GLUON_DEBUG=1`` will provide firmware images including debugging symbols usable with GDB or similar tools. Requires a device or target with at least 16 MB of flash space, e.g. `x86-64`. Unset by default.
+
+lua target files
+================
+
+Target definitions were rewritten in Lua, this was required due to the need to deprecate devices. It also offers the option for more flexible tagging of devices in the future. (`#1745 <https://github.com/freifunk-gluon/gluon/pull/1745>`_)
```

Comment:
"Target definitions have been rewritten in Lua; this was necessary to implement the device deprecation feature."